### PR TITLE
Fix remaining links + add webpack example

### PR DESCRIPTION
**Pull request by stevelacy**
- fix links I did not notice in the last PR.
- adds simple webpack example since there are 4 browserify examples.

### CODE REVIEW

File: docs/recipes/simple-webpack-builds.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Simple webpack builds
+
+### Using [webpack-stream](https://github.com/shama/webpack-stream)
+
+```js
+var gulp = require('gulp');
+var webpack = require('webpack-stream');
```
Comment (phated) [inline docs/recipes/simple-webpack-builds.md]: webpack-stream isn't a recommended plugin.  Just use webpack programatic API directly

**Comment by phated** [comment]
Could you split this into 2 PRs?

**Comment by stevelacy** [comment]
@phated reduced to one commit with just the link changes

**Comment by phated** [comment]
Thanks again @stevelacy

**Pull request merged**